WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -386,6 +386,21 @@ <h4>Where does my reduced data go?</h4>
                     </p>
                 </div>
             </section>
+            <section class="help-topic panel panel-default" data-category="question">
+                <div class="panel-heading" id="reduction-statuses">
+                    <h3>What do the reduction statuses mean?</h3>
+                </div>
+                <div class="panel-body">
+                    There are 5 different states a reduction can be in:
+                    <ul>
+                        <li>Queued - The run is waiting to be processed</li>
+                        <li>Processing - The reduction is in progress</li>
+                        <li>Complete - The reduction has been complete and the reduced data is available</li>
+                        <li>Skipped - The run was marked to be skipped and has not been reduced</li>
+                        <li>Error - Something has gone wrong processing the reduction or in the reduction script</li>
```

Comment:
Maybe we could put the support email in the Error explanation or explain how you would go about debugging the error?

Reply:
Good call. I'll add the support link here as well.

Reply:
Turns out, a very good shout. There were loads of places where the support_email was referenced but not included in the page. I have added it and removed the unecessary if blocks as its hard coded. Will also open an issue to create a config of some sort to store all our email address/external urls to be easily changed/added to a page